README.md

```diff
@@ -1 +1,72 @@
-# .github
\ No newline at end of file
+
+## Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+- Main contributor
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
+- Main contributor: @pavlovcik
+
+
+### 2. ubiquibot/permit-generation
+- Service: Permit Generation module
+- Purpuse: The purpose of this module is to isolate the bot's payment capabilities. This can be directly invoked from other plugins
+- Language: TypeScript
+- External Integration: ENS, ethers
+- Database: Supabase
+- Main contributor: @Keyrxng, @pavlovcik
+
+### 3. ubiquibot/assistive-pricing
+- Service: Assistive Pricing module
+- Purpose: **Don't know**
+- Language: Typescript
```

Comment:
```suggestion
```

Language is probably not a useful list item because all of our code is in TypeScript except for our stablecoin smart contracts.